app/controllers/modules/course/module_host.rb

```diff
@@ -0,0 +1,3 @@
+class Course::ModuleHost
+  include Course::CoursesModulesConcern
```

Comment:
Move that module from `controllers/concerns/course/courses_modules_concern.rb` to the `modules/` directory.